### DISCUSSION TITLE
Laravel Pint: How to make formatting of empty classes and methods the same as non-empty ones?

### DISCUSSION THREAD

**Discussion by ulin-evgeny** [laravel/framework#52238] [Q&A]
Here is the code for my `pint.json` file:
```
{
  "preset": "laravel",
  "rules": {
    "not_operator_with_successor_space": false,
    "concat_space": {
      "spacing": "one"
    },
    "class_attributes_separation": {
      "elements": {
        "method": "one"
      }
    },
    "array_syntax": {
      "syntax": "short"
    },
    "list_syntax": {
      "syntax": "long"
    },
    "braces": {
      "position_after_functions_and_oop_constructs": "next"
    }
  }
}
```

The project has this code:
```
namespace App\Providers;

class SomeClass1
{
    public function register(): void
    {
    }
}
```

Laravel Pint changes it to this code:
```
class SomeClass1
{
    public function register(): void {}
}
```

The same goes for the class. The project has this code:
```
class SomeClass2
{
}
```

Laravel Pint changes it to this code:
```
class SomeClass2 {}
```

I do not like it. I want the formatting of empty classes and methods to look the same as non-empty ones. How can I prevent Laravel Pint from removing newlines from empty methods and classes?

**Comment by apih** [comment]
The formatting probably caused by the change in this [PR](https://github.com/laravel/pint/pull/277). Setting `single_line_empty_body` fixer to `false` would work, if I am not mistaken.
